components/style/values/computed/border.rs

```diff
@@ -81,3 +87,59 @@ impl ToAnimatedZero for BorderCornerRadius {
         Err(())
     }
 }
+
+/// https://drafts.csswg.org/css-backgrounds/#the-borderimage-repeat
+#[derive(Clone, Debug, MallocSizeOf, PartialEq)]
+pub struct BorderImageRepeat(RepeatKeyword, RepeatKeyword);
+
+impl BorderImageRepeat {
+    pub fn repeat() -> Self {
+        BorderImageRepeat(RepeatKeyword::Repeat, RepeatKeyword::Repeat)
+    }
+}
+
+impl ToCss for BorderImageRepeat {
+    fn to_css<W>(&self, dest: &mut W) -> fmt::Result
+    where
+        W: fmt::Write,
+    {
+        match (self.0, self.1) {
+            (horizontal, vertical) => {
+                horizontal.to_css(dest)?;
+                if horizontal != vertical {
+                    dest.write_str(" ")?;
+                    vertical.to_css(dest)?;
+                }
+                Ok(())
+            },
+        }
+    }
+}
+
+impl ToComputedValue for SpecifiedBorderImageRepeat {
+    type ComputedValue = BorderImageRepeat;
+
+    #[inline]
+    fn to_computed_value(&self, _: &Context) -> Self::ComputedValue {
+        match *self {
+            SpecifiedBorderImageRepeat::Repeat => {
+                BorderImageRepeat(RepeatKeyword::Repeat, RepeatKeyword::Repeat)
+            }
+            SpecifiedBorderImageRepeat::Keywords(horizontal, vertical) => {
+                BorderImageRepeat(horizontal, vertical.unwrap_or(horizontal))
+            }
+        }
+    }
+
+    #[inline]
+    fn from_computed_value(computed: &Self::ComputedValue) -> Self {
+        match (computed) {
```

Comment:
Thid doesn't build does it?

Reply:
This is probably still not building, let me know if you need any help setting up the build environment or if I can help somehow, I'd be glad to :)